src/axom/inlet/inlet_utils.cpp

```diff
@@ -17,37 +17,43 @@ void setWarningFlag(axom::sidre::Group* root)
   }
 }
 
-void setRequired(axom::sidre::Group& target, axom::sidre::Group& root, bool required)
+void setFlag(axom::sidre::Group& target,
+             axom::sidre::Group& root,
+             const std::string& flag,
+             bool value)
 {
-  if(target.hasView("required"))
+  if(target.hasView(flag))
   {
     const std::string msg =
-      fmt::format("[Inlet] Required value has already been defined for: {0}",
+      fmt::format("[Inlet] '{0}' value has already been defined for: {1}",
+                  flag,
                   target.getName());
 
     SLIC_WARNING(msg);

```

Comment:
Should it still be a warnable offense if `target.hasView(flag)`  and its value is `value` ?